src/Payments/GPay.res

```diff
@@ -79,6 +80,11 @@ let make = (
     )
   }
 
+  React.useEffect(() => {
```

Comment:
Instead of this, could you please try to make this work with calling **useHandlePostMessages** since this will also help us to get logs

Reply:
Updated